src/frequenz/client/microgrid/_client.py

```diff
@@ -122,3 +125,31 @@ async def __aexit__(
                 "Error while disconnecting from the microgrid API", exceptions
             )
         return result
+
+    async def get_microgrid_info(  # noqa: DOC502 (raises ApiClientError indirectly)
+        self,
+    ) -> MicrogridInfo:
+        """Fetch the information about the local microgrid.
+
+        This consists of information that describes the overall microgrid, as opposed to
+        its electrical components or sensors, e.g., the microgrid ID, location.
+
+        Returns:
+            The information about the local microgrid.
+
+        Raises:
+            ApiClientError: If the are any errors communicating with the Microgrid API,
+                most likely a subclass of
+                [GrpcError][frequenz.client.microgrid.GrpcError].
+        """
+        print(f"CALLED get_microgrid_info(): {self.stub.GetMicrogridMetadata=}")
```

Comment:
This would be an improvement, but could you use a debug log instead?

Reply:
This is an overlook, I will remove the prints!